mod_form.php

```diff
@@ -138,6 +151,35 @@ public function definition() {
             $mform->addElement('static', 'additionalactivitiestwo', '', get_string('additionalactivitiestwo', 'accredible'));
         }
 
+        // Load Accredible attribute keys.
+        $attributekeys = $attributekeysclient->get_attribute_keys();
```

Comment:
We should query the list of attribute keys with `kind: “text”`

Reply:
I have added that parameter as a default one when requesting `attribute_keys`.